services/feed-manager-service/feed-manager-controller/src/main/java/com/thinkbiganalytics/feedmgr/rest/controller/FeedRestController.java

```diff
@@ -910,17 +908,42 @@ public Response uploadFile(@PathParam("feedId") String feedId,
 
         } catch (AccessDeniedException e) {
             String errTemplate = "Permission denied attempting to write file [%s] to [%s]. Check with system administrator to ensure this application has write permissions to folder";
-            String err = String.format(errTemplate, fileMetaData.getFileName(), dropzone);
+            String err = String.format(errTemplate, originalFileName, context.getDropzone());
             log.error(err);
             throw new InternalServerErrorException(err);
 
         } catch (Exception e) {
             String errTemplate = "Unexpected exception writing file [%s] to [%s].";
-            String err = String.format(errTemplate, fileMetaData.getFileName(), dropzone);
+            String err = String.format(errTemplate, originalFileName, context.getDropzone());
             log.error(err);
             throw new InternalServerErrorException(err, e);
         }
-        return Response.ok("").build();
+    }
+
+    private FileUploadContext getFileUploadContext(List<NifiProperty> properties) {
+        String dropzone = null;
+        String regexFileFilter = null;
+
+        for (NifiProperty property : properties) {
+
+            if (property.getProcessorType().equals("org.apache.nifi.processors.standard.GetFile")) {
+                if (property.getKey().equals("File Filter")) {
+                    regexFileFilter = property.getValue();
+                } else if (property.getKey().equals("Input Directory")) {
+                    dropzone = property.getValue();
+                }
+            }
+        }
+
+        return new FileUploadContext(dropzone, regexFileFilter);
+    }
+
+    private List<NifiProperty> getNifiProperties(@PathParam("feedId") String feedId) {
```

Comment:
Please remove `@PathParam` as it has no meaning here.

Reply:
done

Reply:
done